SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Supported Versions
+
+We support only the latest version, and we use [SemVer](https://semver.org/) for
+versioning which means minor version are backwards compatible.
```

Comment:
We should add a sentence here that as long as we are on a `0.x` release, there are no BC guarantees whatsoever. We are not planning to break BC, but it can happen without deprecation cycle even if we are still just upping the minor version. SemVer will only really kick in after the `1.0` release.